src/shared/components/driverAnnotations/DriverAnnotationControls.tsx

```diff
@@ -244,69 +245,74 @@ export default class DriverAnnotationControls extends React.Component<
                                         </label>
                                     </div>
                                 )}
-                            {this.props.handlers
-                                .onChangeAnnotateCBioPortalInputValue && (
-                                <div className="checkbox">
-                                    <label>
-                                        <input
-                                            type="checkbox"
-                                            value={EVENT_KEY.annotateCBioPortal}
-                                            checked={
+                            {getServerConfig().show_cbioportal &&
+                                this.props.handlers
```

Comment:
odd that it needs to check for existnence of the onChange handler.  is it really possible that it wouldn't exist?

Reply:
@alisman i think this is related to reusing this controls component in oncoprint and in oncoprinter